core/agent_state.py

```
@@ -107,4 +122,175 @@ def to_dict(self) -> Dict:
+    
+    def should_perform_gc(self) -> bool:
+        """
+        判断是否应该执行垃圾回收
+        
+        Returns:
+            是否应该执行垃圾回收
+        """
+        return self.step_count > 0 and self.step_count % self.gc_interval == 0
+    
+    def perform_gc(self) -> None:
+        """执行垃圾回收，清理旧的记录"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)